aries-site/src/data/color.js

```diff
@@ -11,8 +11,8 @@ const { layer } = theme;
 /* Color Palette */
 const primaryColors = [
   {
-    name: 'green',
-    value: 'green!',
+    name: 'brand',
+    value: 'brand',
     hex: colors['green!'],
```

Comment:
```suggestion
    hex: colors['brand'],
```
Wondering if we want to change this too just for consistency? Doesn't affect the HEX value, so I'm fine either way.

Reply:
Confirmed that we need to leave as `green!`. `brand` will return and display the string `green!`